truss/util/path.py

```diff
@@ -75,3 +99,65 @@ def build_truss_shadow_target_directory(stub: str, truss_path: Path) -> Path:
     target_directory_path = Path(Path.home(), ".truss", "models", f"{stub}-{suffix}")
     target_directory_path.mkdir(parents=True, exist_ok=True)
     return target_directory_path
+
+
+def load_trussignore_patterns(truss_ignore_file: Path):
+    """Load patterns from a .truss_ignore file"""
+    patterns = []
+    with truss_ignore_file.open("r") as f:
```

Comment:
very minor, please feel free to ignore: I think "r" is the default and can be skipped.